docs/tutorial.rst

```diff
@@ -407,6 +407,225 @@ filter as such in the view:
 
             return qs
 
+Renaming forwarded values
+-------------------------
+- Example source code: `test_project/rename_forward
+  <https://github.com/yourlabs/django-autocomplete-light/tree/master/test_project/rename_worward>`_.
+- Live demo: `Admin / Rename Forward/ Add
+  <http://dal-yourlabs.rhcloud.com/admin/rename_forward/testmodel/add/>`_.
+
+Let's assume that you have the following form using linked autocomplete fields:
+
+.. code-block:: python
+
+    class ShippingForm(forms.Form):
+        src_continent = forms.ModelChoiceField(
+            queryset=Continent.objects.all(),
+            widget=autocomplete.ModelSelect2(url='continent-autocomplete'))
+        src_country = forms.ModelChoiceField(
+            queryset=Country.objects.all(),
+            widget=autocomplete.ModelSelect2(
+                url='country-autocomplete',
+                forward=('src_continent',)))
+        src_city = forms.ModelChoiceField(
+            queryset=City.objects.all(),
+            widget=autocomplete.ModelSelect2(
+                url='city-autocomplete',
+                forward=('src_country',)))
+        # More source fields...
+
+        dst_continent = forms.ModelChoiceField(
+            queryset=Continent.objects.all(),
+            widget=autocomplete.ModelSelect2(url='continent-autocomplete'))
+        dst_country = forms.ModelChoiceField(
+            queryset=Country.objects.all(),
+            widget=autocomplete.ModelSelect2(
+                url='country-autocomplete',
+                forward=('dst_continent',)))
+        dst_city = forms.ModelChoiceField(
+            queryset=City.objects.all(),
+            widget=autocomplete.ModelSelect2(
+                url='city-autocomplete',
+                forward=('dst_country',)))
+        # More destination fields...
```

Comment:
Like limiting fields to country/city? Or to only one continent/country/city field set? Or both?